lib/_stream_readable.js

```diff
@@ -395,9 +395,7 @@ function emitReadable(stream) {
     debug('emitReadable', state.flowing);
     state.emittedReadable = true;
     if (state.sync)
-      process.nextTick(function() {
-        emitReadable_(stream);
-      });
+      process.nextTick(emitReadable_)._setArgs(stream);
```

Comment:
This *may* prove problematic for porting it to readable-stream, since it's using an API that browserify (almost certainly) does not support.